ui/src/components/Custodian/CreateDeposit.css

```diff
@@ -4,11 +4,13 @@
         align-items: flex-start;
     }
 
-    .ui.form > .fields.inline-form-group .field .create-deposit-quantity {
-        padding-left: 5px;
-    }
 
     .custodian-clients .ui.form > .fields.inline-form-group .contract-select-container {
-        padding:  12px 0 12px .5em;
+        padding:  12px 0 12px 0.5em;
```

Comment:
👍

Reply:
Ha, I was experimenting with getting rid of it and I guess I put it back in with the `0`...I really don't have an opinion on zeros in CSS 😄 

And the trailing whitespace removals were just by reflex...